data/src/test/java/org/apache/iceberg/data/DeletesReadTest.java

```diff
@@ -148,15 +144,15 @@ public void testMixedPositionAndEqualityDeletes() throws IOException {
     );
 
     DeleteFile eqDeletes = FileHelpers.writeDeleteFile(
-        table, Files.localOutput(temp.newFile()), Row.of(0), dataDeletes, dataSchema);
+        table, Files.localOutput(temp.newFile()), TestHelpers.Row.of(0), dataDeletes, dataSchema);
```

Comment:
Can you import this class directly to avoid so many changes in this file?

Reply:
Done.